src/main/java/Duke/DukeRobot.java

```diff
@@ -0,0 +1,133 @@
+package Duke;
+
+import Duke.DukeCommandLine.DukeCommandLineInput;
+import Duke.DukeCommandLine.DukeTaskInputException;
+import Duke.DukeFunction.DukeList;
+import Duke.DukeFunction.DukePrinter;
+import Duke.DukeTask.DukeDeadline;
+import Duke.DukeTask.DukeEvent;
+import Duke.DukeTask.DukeTask;
+
+import java.util.Scanner;
+
+public class DukeRobot {
```

Comment:
I notice that you have a tendency to use magic strings and numbers. Do try to avoid that where possible, and instead use constants to set your strings/numbers.